bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/HtmlPrinter.java

```diff
@@ -185,13 +156,11 @@ public static String printParameterSummary(final Altair altair) {
     }
 
     public static String printSummary(final Gems gems) {
-        final FormatStringWriter device = new FormatStringWriter();
-        device.setPrecision(3);
-        String s = "r0(" + gems.getWavelength() + "nm) = " + device.toString(gems.getr0()) + " m\n";
-        s += "Average Strehl = " + device.toString(gems.getAvgStrehl() * 100) + "%\n";
+        String s = String.format("r0(" + gems.getWavelength() + "nm) = %.3f m\n", gems.getr0());
```

Comment:
Again, why not a `StringBuilder` here instead of all these string `+=` inefficient ops?